bin/create-scxa-analytics-suggesters.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+. ${DIR}/schema-version.env
+
+# On developers environment export SOLR_HOST and export SOLR_COLLECTION before running
+HOST=${SOLR_HOST:-"localhost:8983"}
+COLLECTION=${SOLR_COLLECTION:-"scxa-analytics-v${SCHEMA_VERSION}"}
+
+#############################################################################################
+
+printf "\n\nDelete search component for suggesters"
+CURL -X POST -H 'Content-Type: application/json' -d '{
```

Comment:
Why is curl capitalized here? should be just CURL unless that you are adding a wrapper for CURL somewhere that I'm missing? I'm surprised that tests are passing with this.

Reply:
Ohh, this is passing tests because this is not including any tests yet.

Reply:
I would comment here too, please use `curl` instead for passing tests.

Reply:
copy and paste error :). I fixed it.